webpack.mix.js

```diff
@@ -20,7 +20,7 @@ mix.webpackConfig({
         ],
         resolve: {
             alias: {
-                Horizon: path.resolve(__dirname, 'vendor/laravel/horizon/resources/js/')
+                Horizon: path.resolve(__dirname, 'vendor/laravel/horizon/resources/assets/js/')
```

Comment:
Is this part of the this PR? There is another ticket for Horizon.

Reply:
This can be removed - we brought it in via another ticket.